ares/urls.py

```diff
@@ -3,8 +3,10 @@
 """
 from django.conf.urls import url, include
 from django.contrib import admin
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url('^accounts/', include('accounts.urls')),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
I understand your security standpoint but I put it here for following reasons. If you're not convinced at the end I can add `static()` in DEBUG mode like this:
```
if settings.DEBUG:
    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
However my reasons are:
1. We want to be able to serve other media files (any other user uploaded content like their profile picture maybe).
2. Serving `code` files to their owners also should be ok. (right?)
3. We need file serving functionality in development environment without a reverse proxy.
4. We will be using a reverse proxy in production in front of Django (i.e Nginx)

With these in mind we can use one of these two solutions:
* Use X-Sendfile (X-Accel on Nginx) to handle file serving authorization efficiently. For reference, please take a look at [Official Nginx Docs](https://www.nginx.com/resources/wiki/start/topics/examples/x-accel/) or [this post](https://wellfire.co/learn/nginx-django-x-accel-redirects/) or [django-sendfile](https://github.com/johnsensible/django-sendfile)
* Save `code` files to a location other than MEDIA_ROOT (by doing this serving them to their owners needs extra care)

P.S I suggest the former